tests/console/console.go

```diff
@@ -119,7 +121,9 @@ func VmiConsoleRunCommand(vmi *v1.VirtualMachineInstance, command string, timeou
 // It will parse the kernel output (dmesg) and succeed if it finds that Secure boot is enabled
 func SecureBootExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, error) {
 	virtClient, err := kubecli.GetKubevirtClient()
-	PanicOnError(err)
+	if err != nil {
+		return nil, err
```

Comment:
let's add some error wrapping here so it's easier to debug.

Reply:
More or less something like  "failed starting secure boot expecter: %w" so we know wich one of all the `GetKubevirtClient` calls is failing, at the end is a poor's man stack trace.

Reply:
I understand your point, this is something we should probably take as a pattern in the tests.
But in this specific case, `GetKubevirtClient()` will fail on the first occurrence in the suite setup and never in other places.